test/integration/toggle-notes.spec.js

```diff
@@ -35,4 +35,22 @@ describe('Toggle Scribe Notes', function(){
       });
     });
   });
+
+
+  given('we have on expanded note', function(){
+    givenContentOf('<p>On the 24th of February, <gu-note class="note note--start note--end">1815, the</gu-note> look-out at Notre-Dame de la Garde signalled the three-master, the Pharaon from Smyrna</p>', function() {
```

Comment:
I think we should remove the lines from Monte Cristo and use some content that makes sense to the test